File: lab05/Triangle.java
// File: Triangle.java
// Author: Dr. Watts
// Contents: This file contains the description and implementation
// of a class called Triangle. 

public class Triangle extends Shape
{
	Triangle ()
	{
	}

	public String sides ()
	{
		return "3";
	}
	
	public String getName ()
	{
		return "Triangle";
	}

	public static void main (String [] args)
	{
		Triangle T = new Triangle ();
		T.write ();
		System.out.println ();
	}
}

File: lab04/Triangle.cpp
// File: Triangle.cpp
// Author: Amandeep Gill
// Contents: This file contains the implementation for the Triangle Class

#include <string>
#include "Triangle.h"

using namespace std;

Triangle::Triangle()
{
    debug << "Entering default constructor for Triangle\n";
    side = 0;
    debug << "Exiting default constructor for Triangle\n";
}

string Triangle::Sides() const
{
    debug << "Entering Sides function for Triangle\n";
    debug << "Exiting Sides function for Triangle\n";
    return "3";
}

